### PR TITLE
test: AgencyConsistencyValidator

### PR DESCRIPTION
**Pull request by lionel-nj**
closes #559 
**Summary:**

This PR adds unit tests for existing validation rules

**Expected behavior:** 

No code change, tests should pass

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `gradle test` to make sure you didn't break anything
- [x] Format the title like "feat: -new feature short description-" (PR title must follow the [conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/))
- [x] Linked all relevant issues
- ~[ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)~
- [x] Check copyright headers
- [x] Check no wildcards in import statements

### CODE REVIEW

File: main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidatorTest.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    AgencyConsistencyValidator underTest = new AgencyConsistencyValidator();
+    underTest.agencyTable =
+        createAgencyTable(
+            noticeContainer,
+            ImmutableList.of(
+                createAgency(
+                    0,
+                    "first agency",
+                    "agency name",
+                    "url value",
```
Comment (aababilov) [inline main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidatorTest.java:68]: Nit: let's make all of them valid URLs, e.g., http://agency1.com/

**Review by aababilov — APPROVED** [review]
Thank you!

File: main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidatorTest.java (hunk 1 above)
Comment (lionel-nj) [reply main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidatorTest.java:68]: Agree, if you do not mind I'd suggest using a url that MobilityData has control over: www.mobilitydata.org

Comment (lionel-nj) [reply main/src/test/java/org/mobilitydata/gtfsvalidator/validator/AgencyConsistencyValidatorTest.java:68]: Done.

**Pull request merged**